packages/server-wallet/e2e-test/challenge.test.ts

```diff
@@ -55,7 +55,9 @@ afterAll(async () => {
   await knexPayer.destroy();
 });
 
-test('the engine handles the basic challenging v0 behavior', async () => {
+// TODO: This test relies on the engine being tightly coupled to the chain service.
+// This should be refactored to use the wallet once the challenge method is exposed there.
+test.skip('the engine handles the basic challenging v0 behavior', async () => {
```

Comment:
We should probably create an issue for this.

Reply:
Can you create issues for the lost test coverage? The skipped test here is about challenging. On this branch, we now have a great directly funded channel test, but challenging isn't tested e2e

Reply:
that is, _assuming_ we merge #3587